docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -96,12 +96,19 @@ spec:
           threads: 2
           cores: 5
           dedicatedCpuPlacement: true
+          isolateEmulatorThread: true
 ...
 ```
 > **Notes**
 > - The sockets field should be 1, in order to make sure the CPUs are scheduled from the same NUMA Node.
 > - In this snippet example, the VM will be scheduled with 5 [hyper-threads](https://access.redhat.com/articles/7445) (or 10 CPUs).
 
+#### SMT Aligment
+On Nodes where SMT is enabled and [CPU-Manager's policy](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy-options) is set to `full-pcpus-only`, there is a significance to the amount of CPUs requested. If the requested total CPUs is even, the isolateEmulatorThread feature requires also doing the following in order to avoid `SMT Alignment` errors by setting the [alignCPUs](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#aligncpus-feature-gate) enabler on HCO:
```

Comment:
```suggestion
On Nodes where SMT is enabled and CPU-Manager's policy is set to [full-pcpus-only](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy-options) , in order to avoid `SMT Alignment` errors - set the [alignCPUs](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#aligncpus-feature-gate) enabler on HCO:
```

Reply:
Like, DONE